src/lib/fastDownward/fastDownward.ts

```diff
@@ -30,6 +31,7 @@ export function search<LU extends LearningUnit>({
 }) {
     // Non recursive part of the search
     const globalKnowledge = new GlobalKnowledge(allSkills);
+    const skillsRelations = new SkillsRelations(allSkills);
```

Comment:
Consider merging globalKnowledge and SkillRelations into one class.

Reply:
I merged both classes into one class called GlobalKnowledge.